src/test/java/myfirstproject/service/impl/PrepareDataImplTest.java

```diff
@@ -0,0 +1,33 @@
+package myfirstproject.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
here should be 
`import static org.junit.Assert.assertEquals;`
check everywhere (just reimport methods)

Reply:
but if we do not have this dependency in pom.xml
then we can't make import . just look:

![image](https://user-images.githubusercontent.com/34911404/201147261-b6ccc437-f7c5-4de6-928e-15f0141a2ba6.png)

Reply:
okay... I did it.